src/QtLocationPlugin/ElevationMapProvider.h

```diff
@@ -6,25 +6,29 @@ static constexpr const quint32 AVERAGE_AIRMAP_ELEV_SIZE = 2786;
 
 class ElevationProvider : public MapProvider
 {
-    Q_OBJECT
-
 protected:
-    ElevationProvider(const QString& imageFormat, quint32 averageSize,
-                      QGeoMapType::MapStyle mapType, QObject* parent = nullptr)
-        : MapProvider(QStringLiteral("https://terrain-ce.suite.auterion.com/"), imageFormat, averageSize, mapType, parent) {}
+    ElevationProvider(const QString &mapName, const QString &imageFormat, quint32 averageSize,
+                      QGeoMapType::MapStyle mapType)
+        : MapProvider(
+            mapName,
+            QStringLiteral("https://terrain-ce.suite.auterion.com/"),
```

Comment:
This was already the case, but since you're changing it, I would pass the url from `CopernicusElevationProvider`